.github/workflows/ci-ipfs.yml

```diff
@@ -54,30 +56,47 @@ jobs:
           BLUMEN_W3S_TOKEN: ${{ secrets.W3S_TOKEN }}
           BLUMEN_GW3_TOKEN: ${{ secrets.GW3_TOKEN }}
           PATH_TO_IPFS_CONTENT: ${{ inputs.ipfs_folder }}
-      - name: Zip
+      - name: Create ipfs source code archive
         if: inputs.is_create_release
-        id: zip
+        id: car
         run: |
-          zip -r $FILE_NAME $PATH_TO_IPFS_CONTENT
+          mv /tmp/$PATH_TO_IPFS_CONTENT.car ./$FILE_NAME.car
+          zip -r $FILE_NAME.zip $PATH_TO_IPFS_CONTENT/
         env:
-          PATH_TO_IPFS_CONTENT: "${{ inputs.ipfs_folder }}/"
+          PATH_TO_IPFS_CONTENT: ${{ inputs.ipfs_folder }}
+          FILE_NAME: ${{ env.FILE_NAME }}
       - name: Set output tag and date
         if: inputs.is_create_release
         id: vars
         run: |
-          echo "TAG=${GITHUB_REF#refs/*/}" >> $GITHUB_OUTPUT
+          echo "TAG=`echo $(git describe --abbrev=0 --tags)`" >> $GITHUB_OUTPUT
           echo "DATE=$(date +'%Y-%m-%d')" >> $GITHUB_OUTPUT
+      - name: Get release body from existing release
+        if: inputs.is_create_release
+        id: body
+        run: |
+          export EXISTING_BODY=`gh release view $TAG --json body | jq -r '.body'`
+          export EXISTING_TITLE=`gh release view $TAG --json name | jq -r '.name'`
+          echo "EXISTING_BODY<<EOF" >> $GITHUB_OUTPUT   
+          echo "$EXISTING_BODY" >> $GITHUB_OUTPUT   
+          echo "EOF" >> $GITHUB_OUTPUT
+          echo "EXISTING_TITLE<<EOF" >> $GITHUB_OUTPUT
+          echo "$EXISTING_TITLE" >> $GITHUB_OUTPUT
+          echo "EOF" >> $GITHUB_OUTPUT
```

Comment:
you can write it to the file and then just use `--notes-file `, maybe it would looks better